src/riak_kv_wm_timeseries.erl

```diff
@@ -399,14 +399,18 @@ extract_field_value({Name, Type, Optional}, FVList) ->
             throw({missing_field, Name});
         undefined when Optional == true ->
             [];
-        [] when Optional == true ->
```

Comment:
Removing this case *may* be a change in behaviour for the degenerate HTTP usage that we *may* have previously allowed, specifying `null` for a JSON field using `''`. I did not find any documentation encouraging or suggesting to use `''` for `null`.